Celeste.Mod.mm/Patches/OuiMainMenu.cs

```diff
@@ -9,6 +9,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Runtime.CompilerServices;
```

Comment:
```suggestion
```
Looks like an accidental using.

Reply:
Addressed in commit [a87a04f](https://github.com/EverestAPI/Everest/pull/836/commits/a87a04fdafc78728fdb24bff92cbe487f25b805b)